examples/contracts/generics_forwarded/src/generic.rs

```diff
@@ -0,0 +1,89 @@
+use cosmwasm_std::{CosmosMsg, CustomMsg, Response, StdError, StdResult};
+use generic::Generic;
+use serde::de::DeserializeOwned;
+use serde::Deserialize;
+use sylvia::contract;
+use sylvia::types::{CustomQuery, ExecCtx, QueryCtx, SvCustomMsg};
+
+#[contract(module = crate::contract)]
+#[messages(generic as Generic)]
+#[sv::custom(msg=CustomMsgT, query=CustomQueryT)]
+impl<InstantiateT, ExecT, QueryT, MigrateT, CustomMsgT, CustomQueryT, FieldT>
+    Generic<ExecT, QueryT, SvCustomMsg>
+    for crate::contract::GenericsForwardedContract<
+        InstantiateT,
+        ExecT,
+        QueryT,
+        MigrateT,
+        CustomMsgT,
+        CustomQueryT,
+        FieldT,
+    >
+where
+    for<'msg_de> InstantiateT: CustomMsg + Deserialize<'msg_de> + 'msg_de,
+    ExecT: CustomMsg + DeserializeOwned + 'static,
+    QueryT: CustomMsg + DeserializeOwned + 'static,
+    MigrateT: CustomMsg + DeserializeOwned + 'static,
+    CustomMsgT: CustomMsg + DeserializeOwned + 'static,
+    CustomQueryT: CustomQuery + 'static,
+    FieldT: 'static,
+{
+    type Error = StdError;
+
+    #[msg(exec)]
+    fn generic_exec(&self, _ctx: ExecCtx, _msgs: Vec<CosmosMsg<ExecT>>) -> StdResult<Response> {
+        Ok(Response::new())
+    }
+
+    // Sylvia will fail if single type is used to match against two different generics
+    // It's because we have to map unique generics used as they can be used multiple times.
+    // If for some reason like here one type would be used in place of two generics either full
```

Comment:
There are plenty of reasons for that, and we cannot make people using aliases for every generic String.

Reply:
These comments are actually related to the `generic_contract` where we pass the concrete types to the generics.
Comment landed here when I copied the examples.